core/src/test/java/nl/nn/adapterframework/util/SizeLimitedVectorTest.java

```diff
@@ -0,0 +1,55 @@
+package nl.nn.adapterframework.util;
+
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+/**
+ * SizeLimitedVector Tester.
+ *
+ * @author <Sina Sen>
+
+ */
+public class SizeLimitedVectorTest {
+
+
+    /**
+     *
+     * Method: add(Object o)
+     *
+     */
+    @Test
+    public void testAdd() throws ArrayIndexOutOfBoundsException {
```

Comment:
You dont have to declare this RuntimeException here do you? Non of the functions are throwing any exceptions

Reply:
Yes. I am going to remove it, thanks for pointing out.